libs/wingsdk/src/target-tf-gcp/app.ts

```diff
@@ -31,7 +31,7 @@ export interface AppProps extends CdktfAppProps {
    * The Google Cloud zone, used for all resources.
    * @see https://cloud.google.com/functions/docs/locations
    */
-  readonly zone: string;
+  readonly zone?: string;
```

Comment:
This change seems a little out of place, why was it added?

Reply:
It was needed because there is code that doesn't pass `zone` and it failed complication. Not sure exactly how our main build passed.